MekHQ/src/mekhq/gui/model/UnitTableModel.java

```diff
@@ -97,7 +97,7 @@ public String getColumnName(int column) {
             case COL_FORCE -> "Force";
             case COL_CREW -> "Crew";
             case COL_TECH_CRW -> "Tech Crew";
-            case COL_MAINTAIN -> "Maintenance Costs";
+            case COL_MAINTAIN -> "Maintenance";
```

Comment:
This helps reduce the amount of space required to fully display this column's information.